Havit/Havit/Screens/CategoryContents/View/CategoryPanModalViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  CategoryPanModealViewController.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/19.
+//
+import UIKit
+
+import PanModal
+import SnapKit
+
+class CategoryPanModalViewController: BaseViewController, PanModalPresentable {
+    var panScrollable: UIScrollView? {
+           return nil
+    }
+    
+    var shortFormHeight: PanModalHeight {
+        return .contentHeight(400)
+    }
+
+    var longFormHeight: PanModalHeight {
+        return .contentHeight(400)
+    }
+    
+    var cornerRadius: CGFloat {
+        return 0
+    }
```

Comment:
p3;

혹시 
```suggestion
    var panScrollView = UIScrollView()
    var shortFormHeight: PanModalHeight = .contentHeight(400)
    var longFormHeight: PanModalHeight = .contentHeight(400)
    var cornerRadius: CGFloat = 0
```

이렇게 쓰는건 안될까유?

Reply:
f7243580